src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -64,6 +64,15 @@ public class ConvertToJsonCommand : PSCmdlet
         [Parameter()]
         public SwitchParameter EnumsAsStrings { get; set; }
 
+        /// <summary>
+        /// gets or sets the AsArray property.
```

Comment:
Please start the public comment with capital letter.

Reply:
Fixed.